tests/sm_int/sm_int.c

```diff
@@ -526,12 +494,14 @@ BEGIN_TEST_SUITE(sm_int_tests)
 TEST_SUITE_INITIALIZE(suite_init)
 {
     ASSERT_ARE_EQUAL(int, 0, gballoc_hl_init(NULL, NULL));
-    GetSystemInfo(&systemInfo);
-    dwNumberOfProcessors = systemInfo.dwNumberOfProcessors;
-    dwNumberOfProcessors = 2;
-    ASSERT_IS_TRUE(dwNumberOfProcessors * 4 <= N_MAX_THREADS, "for systems with maaany processors, modify N_MAX_THREADS to be bigger");
+#if _MSC_VER
+    numberOfProcessors = sysinfo_get_processor_count();
```

Comment:
>sysinfo_get_processor_count [](start = 25, length = 27)

I thought you added this to PAL for both platforms

Reply:
I did, but for Linux we still need to use less procs and smaller array sizes because Helgrind and DRD take too long otherwise

---
In reply to: [516820129](https://github.com/Azure/c-util/pull/75#discussion_r516820129) [](ancestors = 516820129)